src/styles/global/forms/forms.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:math';
+
```

Comment:
Please remove no longer needed `@use 'sass:math';` since we use Bootstrap 4.6.1.